app/models/goldencobra/upload.rb

```diff
@@ -97,17 +97,18 @@ def unzip_files
       if self.image_file_name.include?(".zip") && File.exists?(self.image.path)
         require 'zip'
         zipped_files = Zip::File.open(self.image.path)
-        int = 0
         zipped_files.each do |zipped_file|
-          int = int + 1
+          filename = zipped_file.name.split('/').last.gsub(" ", "_")
```

Comment:
Could there be other chars we need to escape? Could something like `.parameterize` be helpful here?

Reply:
parameterize removes "." - if this is okay I will add it instead of the gsub